kv/kvserver: TestReplicateQueueRebalanceMultiStore failed

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15565041?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15565041?buildTab=artifacts#/) on master @ [c7f37e813dbd1e6d2afdb88c5e9cd986dc061299](https://github.com/cockroachdb/cockroach/commits/c7f37e813dbd1e6d2afdb88c5e9cd986dc061299):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [18 62 16 31 29 34 42 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 59 9 26 26 29 36 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 28 44 19 27 25 29 45]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [21 31 28 44 30 34 35 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 39 16 57 32 42 45 4]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 31 26 35 30 39 39 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 29 39 20 29 30 4 70]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 32 38 16 30 29 6 72]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 33 27 33 29 29 45 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 32 24 39 29 30 44 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 32 46 22 37 32 20 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 32 43 20 39 32 20 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 31 4 61 27 33 43 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 29 47 26 28 27 39 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 29 68 12 35 27 25 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 29 31 51 35 27 28 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 28 19 60 27 34 32 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 31 53 20 25 36 26 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 34 60 17 34 24 27 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 30 23 58 41 19 31 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 22 9 65 19 51 28 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 22 26 50 18 51 27 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 26 53 20 33 33 24 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 28 38 41 39 28 27 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 29 36 42 31 30 29 28]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [11 11 12 9 0 14 11 16]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 36 29 32 43 24 31 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 41 35 7 51 23 28 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 23 28 35 38 33 32 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 23 26 49 25 38 32 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 38 38 29 41 27 29 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 44 42 13 48 16 31 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 28 31 39 34 26 32 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 20 6 71 8 60 33 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [18 36 41 37 35 34 30 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [18 42 46 34 38 28 26 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [56 12 34 29 33 32 36 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [62 6 29 29 27 36 38 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 42 29 30 25 37 34 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [20 54 32 30 28 32 32 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [49 23 30 29 20 46 33 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [50 18 26 29 5 59 39 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [20 40 29 27 22 55 37 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [13 50 36 30 65 15 19 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 26 33 28 37 34 38 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [50 18 30 27 16 48 49 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 28 27 27 30 34 48 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 30 27 27 35 29 42 33]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [36 30 27 27 35 29 42 33]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (245.51s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-39408

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15565184?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15565184?buildTab=artifacts#/) on master @ [c7f37e813dbd1e6d2afdb88c5e9cd986dc061299](https://github.com/cockroachdb/cockroach/commits/c7f37e813dbd1e6d2afdb88c5e9cd986dc061299):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [47 32 0 30 30 32 79 2]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [46 33 0 37 31 33 60 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [16 32 43 21 33 35 28 51]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 28 53 17 32 29 26 51]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [44 25 24 37 32 26 41 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [10 48 56 9 31 28 25 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [3 53 58 4 34 29 16 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 22 38 34 27 24 20 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [60 8 18 60 22 24 41 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [58 13 22 60 22 27 46 9]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [52 17 23 58 22 25 44 16]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 41 30 36 29 24 23 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 14 48 16 47 33 52 5]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 31 42 28 48 34 44 5]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [14 61 18 45 16 35 2 65]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 39 29 40 29 24 18 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [51 12 39 34 41 23 37 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 46 39 26 31 27 31 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [0 74 22 27 16 42 25 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [0 50 29 40 34 31 26 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [74 1 41 35 51 11 37 4]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [76 1 47 31 55 17 27 2]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [60 17 42 26 27 49 21 14]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [59 18 41 26 26 50 20 16]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 37 27 33 35 28 47 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 27 30 27 35 27 46 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [48 16 41 19 8 65 9 49]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [20 43 43 19 7 54 33 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [15 58 29 46 7 16 75 9]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 36 22 48 38 2 75 9]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 19 26 34 51 7 37 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 19 35 29 42 21 35 46]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [21 41 19 50 9 54 30 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 35 19 51 9 46 31 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [48 15 33 32 51 23 31 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 31 35 30 41 33 27 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [7 70 28 29 15 44 28 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [16 67 30 29 0 46 31 39]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [16 67 31 29 0 45 33 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [19 38 32 30 48 28 30 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [15 38 29 30 48 29 29 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [51 18 32 32 26 32 34 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [53 17 34 33 26 29 35 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [43 35 31 33 31 26 36 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [43 36 30 33 31 29 29 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [15 40 36 30 27 35 42 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [14 41 36 30 27 35 42 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [56 8 29 31 34 30 38 33]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [56 8 29 31 34 30 38 33]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.09s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15565501?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15565501?buildTab=artifacts#/) on master @ [c7f37e813dbd1e6d2afdb88c5e9cd986dc061299](https://github.com/cockroachdb/cockroach/commits/c7f37e813dbd1e6d2afdb88c5e9cd986dc061299):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 27 9 46 43 35 38 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 29 47 28 32 26 28 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 31 63 14 20 25 24 50]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 29 32 40 41 23 22 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [42 30 7 53 50 31 40 3]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 29 40 25 35 33 31 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 29 45 21 29 34 15 61]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 24 30 37 36 20 31 51]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 28 5 51 46 13 54 23]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 29 36 31 38 22 51 20]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 29 38 29 36 28 46 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 29 24 40 25 28 35 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [44 27 3 55 43 8 68 6]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 30 3 56 40 30 52 6]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 29 49 14 27 39 21 45]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 29 38 29 26 39 27 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 32 7 70 42 14 44 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 33 10 44 32 40 44 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 23 51 15 23 49 29 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 41 56 16 33 26 30 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 47 10 71 42 7 34 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 37 6 54 34 40 32 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 24 33 36 21 51 32 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 30 58 19 31 29 31 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 45 3 35 52 11 33 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 17 45 25 33 37 35 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 10 58 18 23 54 34 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 53 40 28 27 17 27 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [16 56 32 32 29 10 32 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [42 38 32 32 28 32 30 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 31 37 35 27 48 23 3]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [9 41 32 35 43 29 30 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [8 47 31 34 50 10 28 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [49 25 32 32 35 14 28 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [50 25 29 31 32 19 32 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 26 34 33 26 32 36 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [15 35 37 34 29 37 36 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 36 30 30 34 39 32 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [50 24 27 31 33 29 28 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [11 48 38 28 14 48 28 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [7 52 41 27 12 69 30 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [59 14 37 28 43 44 27 6]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [76 4 13 26 61 20 17 38]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [53 22 18 28 56 21 18 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 46 32 26 37 26 16 44]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 20 31 38 31 31 31 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [44 7 32 46 29 29 43 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [9 60 29 29 30 37 25 35]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [9 60 29 29 30 37 25 35]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15586155?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15586155?buildTab=artifacts#/) on master @ [e924a27896b9fbf495609d3965612366de6ebf78](https://github.com/cockroachdb/cockroach/commits/e924a27896b9fbf495609d3965612366de6ebf78):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 36 44 28 9 39 32 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 39 15 61 9 31 35 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 38 15 64 9 30 37 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 33 43 41 9 30 35 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 37 38 30 9 41 33 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 41 34 27 9 40 36 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 29 34 30 9 39 40 39]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 30 42 40 9 25 35 39]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 44 39 36 9 32 33 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 44 28 11 9 57 39 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 42 29 27 9 47 40 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 36 17 62 9 18 35 41]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 34 23 57 9 23 33 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 36 42 26 9 49 36 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 27 41 25 9 51 33 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 30 39 26 9 46 36 38]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 30 39 39 9 34 39 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 37 40 35 9 37 37 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 38 41 31 9 42 32 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [51 25 12 63 9 30 34 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [51 27 19 57 9 29 34 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [21 45 67 17 9 37 31 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [18 47 64 17 9 38 33 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 28 36 49 9 36 33 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 34 37 46 9 33 35 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [14 65 37 22 9 40 31 41]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 55 34 26 9 38 31 41]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [56 25 27 42 9 30 34 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [57 25 29 43 9 31 34 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 52 30 32 9 37 31 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [8 74 32 21 9 43 32 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [8 74 32 21 9 43 32 38]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 49 27 26 9 42 32 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [54 31 30 28 9 37 29 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 48 31 34 9 35 31 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [10 52 37 45 9 26 52 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [42 29 30 40 9 34 51 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [75 6 23 31 9 46 19 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [57 23 22 20 9 51 9 67]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 48 24 21 9 43 36 42]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 51 28 29 9 28 62 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [43 28 29 33 9 36 32 49]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 33 35 36 9 48 12 64]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [13 65 30 26 9 36 36 43]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 46 29 29 9 28 61 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 46 28 29 9 29 47 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 31 32 30 9 44 32 50]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 28 35 34 9 42 51 32]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [27 28 35 34 9 42 51 32]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.98s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15586785?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15586785?buildTab=artifacts#/) on master @ [e924a27896b9fbf495609d3965612366de6ebf78](https://github.com/cockroachdb/cockroach/commits/e924a27896b9fbf495609d3965612366de6ebf78):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [8 66 30 27 29 26 19 53]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [18 54 32 25 27 28 19 53]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [55 24 28 29 26 31 38 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 37 27 30 34 30 50 14]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [14 41 32 27 41 27 9 63]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [49 22 26 28 29 33 29 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [55 11 23 32 11 41 73 9]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [50 21 26 30 29 33 66 1]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 47 34 27 51 15 19 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [24 53 37 24 45 18 16 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 36 31 29 18 43 36 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 34 30 30 29 38 40 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 30 31 30 37 35 38 21]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [24 35 34 32 24 30 24 56]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [43 18 30 29 11 46 46 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [12 53 32 31 29 37 49 16]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [4 57 29 30 56 5 3 71]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 27 27 27 39 25 31 52]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [62 3 27 26 4 56 71 7]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [44 19 27 27 17 45 76 2]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [18 45 28 28 42 21 39 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 37 26 27 51 20 16 56]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [52 12 28 29 24 41 35 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 24 32 31 22 38 56 3]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 57 30 30 57 12 32 16]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [19 35 28 30 38 37 27 41]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [19 28 32 33 28 47 29 41]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [42 28 32 32 54 15 27 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [47 28 32 30 61 1 22 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 34 35 31 33 44 22 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [13 25 32 36 3 73 44 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 23 29 34 4 65 44 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 22 30 31 29 46 35 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 32 30 32 64 20 22 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 34 37 29 23 38 47 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [45 18 16 44 38 24 22 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 17 16 49 40 19 0 62]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [11 58 38 31 37 31 0 52]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [8 67 65 12 30 54 0 18]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 34 61 12 23 48 31 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 27 45 28 27 37 32 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 33 28 38 36 30 27 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 36 47 18 36 32 28 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 24 37 37 32 30 33 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 20 21 49 32 31 33 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [21 50 47 12 34 30 29 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 54 54 7 37 32 30 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [46 19 26 46 32 31 28 30]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [46 19 26 46 32 31 28 30]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.04s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15586881?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15586881?buildTab=artifacts#/) on master @ [e924a27896b9fbf495609d3965612366de6ebf78](https://github.com/cockroachdb/cockroach/commits/e924a27896b9fbf495609d3965612366de6ebf78):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [59 16 50 14 13 20 68 15]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [39 24 48 14 28 20 72 11]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 24 45 17 30 19 66 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 29 38 28 29 19 46 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 43 39 35 35 19 29 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [55 14 32 40 31 19 37 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [55 14 32 40 31 19 37 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 38 30 30 29 19 45 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [16 59 38 28 33 19 26 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [54 20 30 33 37 19 32 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [69 7 19 35 28 19 67 11]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 41 22 32 28 19 59 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [4 71 50 27 26 19 9 50]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [11 35 52 29 45 19 3 63]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [80 2 41 41 53 19 3 15]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 46 36 37 35 19 15 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [2 74 39 26 18 19 10 67]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [1 48 31 42 34 19 29 54]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 14 19 43 52 19 55 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [65 1 34 28 47 19 58 4]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [58 12 46 27 47 19 41 7]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 52 44 27 30 19 19 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 55 30 29 34 19 23 42]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [57 13 14 41 39 19 62 13]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [65 15 47 23 18 19 54 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 33 13 45 47 19 34 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [14 30 0 68 58 19 37 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [39 18 31 43 51 19 32 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [46 17 34 42 44 19 28 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [15 59 38 34 27 19 33 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [6 72 18 40 30 19 35 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 48 21 36 36 19 35 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 34 31 31 37 19 35 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [51 27 38 32 35 19 28 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [51 28 35 33 34 19 28 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [12 59 36 28 32 19 40 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [7 62 38 25 32 19 45 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [57 23 29 31 32 19 36 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [61 18 30 36 31 19 32 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 50 35 30 31 19 31 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 52 32 30 34 19 32 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [52 20 27 42 32 19 34 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [55 12 40 39 34 19 25 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 54 39 29 32 19 27 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [2 77 27 21 32 19 48 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 59 27 28 34 19 41 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [71 10 27 43 37 19 18 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [64 15 29 44 34 19 20 32]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [64 15 29 44 34 19 20 32]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15592978?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15592978?buildTab=artifacts#/) on master @ [e09f5356381704e404eba1d771152f73c7c8a572](https://github.com/cockroachdb/cockroach/commits/e09f5356381704e404eba1d771152f73c7c8a572):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 29 22 50 31 31 24 41]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 29 60 13 34 22 20 52]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 28 62 16 26 25 53 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 29 38 34 27 27 53 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 32 28 32 30 29 24 53]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 29 22 27 37 32 18 62]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 28 31 26 30 28 59 21]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 29 38 29 24 26 70 15]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 31 31 32 27 26 33 52]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 29 27 30 32 30 23 58]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 31 41 25 29 30 54 10]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 35 43 28 28 30 57 6]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 31 32 31 29 30 23 57]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 27 25 30 27 29 12 74]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 23 30 28 27 26 55 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 24 30 28 27 27 54 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [21 50 35 28 31 36 34 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 51 36 30 27 45 12 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 26 31 33 29 37 40 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [49 17 31 33 29 25 48 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [24 56 30 32 26 28 33 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [11 78 31 30 31 33 16 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [48 29 29 30 32 30 40 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 32 28 29 28 28 44 33]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [15 16 6 11 6 7 14 9]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [18 38 31 31 29 32 20 56]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [44 22 27 29 28 30 27 51]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [50 16 28 30 28 29 46 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 30 33 32 30 31 44 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 40 33 26 30 31 43 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 42 28 30 31 33 31 31]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [17 16 8 6 11 10 4 12]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 16 31 38 33 31 45 23]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [20 34 39 30 34 29 45 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [9 66 35 19 28 33 23 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 44 27 25 31 32 30 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [43 39 29 33 32 26 37 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 33 36 29 35 28 40 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 33 32 28 31 36 33 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 31 26 29 29 33 35 44]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [44 23 33 37 28 17 59 13]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 24 33 39 31 22 48 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 29 33 34 25 38 22 47]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 33 32 32 28 35 38 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 31 32 31 32 27 43 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 31 32 31 25 33 39 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 31 31 30 22 44 10 57]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 32 31 30 24 41 20 50]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [30 32 31 30 24 41 20 50]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.61s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15599944?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15599944?buildTab=artifacts#/) on master @ [41c37cbcc53d34b01d5070c52158f63bb78edd13](https://github.com/cockroachdb/cockroach/commits/41c37cbcc53d34b01d5070c52158f63bb78edd13):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 29 29 29 26 48 44 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 31 29 29 42 34 36 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 31 29 28 46 27 33 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 34 31 34 20 35 31 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 30 33 29 49 25 33 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 29 33 28 55 23 33 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 30 30 35 32 36 18 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 31 27 33 12 43 21 53]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 31 27 29 39 30 23 46]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 30 30 28 40 30 24 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 32 33 32 26 50 31 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 25 28 36 22 53 31 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 24 26 32 41 31 30 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 36 40 18 53 11 28 41]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 29 25 37 38 38 32 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 26 14 54 15 61 37 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 25 38 35 41 31 27 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 29 58 10 62 8 10 51]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 29 43 21 43 28 26 43]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 29 33 31 32 35 35 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 32 30 32 39 23 25 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 34 30 34 45 15 17 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 32 30 35 30 26 49 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 29 30 33 14 53 62 8]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 29 29 30 26 41 49 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 30 28 24 45 19 19 60]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 27 26 30 34 39 40 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 31 26 36 7 57 50 17]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 30 28 23 52 27 36 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 30 29 21 59 25 26 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 29 27 44 32 37 35 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 35 30 48 4 46 40 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 34 33 34 32 28 44 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 29 26 24 71 9 26 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 28 26 31 19 62 25 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 29 29 41 9 73 34 10]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 31 30 27 45 30 15 48]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 29 27 11 66 9 7 74]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 24 24 37 30 45 37 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 22 19 52 13 60 55 10]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 23 25 33 43 30 44 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 24 28 26 60 14 31 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 28 29 30 33 53 25 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 29 30 31 24 59 40 18]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 31 31 29 42 23 26 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 31 31 30 44 17 26 47]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 31 31 28 21 53 32 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 32 31 28 21 53 34 29]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [30 32 31 28 21 53 34 29]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (245.66s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot T-kv branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15600362?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15600362?buildTab=artifacts#/) on master @ [41c37cbcc53d34b01d5070c52158f63bb78edd13](https://github.com/cockroachdb/cockroach/commits/41c37cbcc53d34b01d5070c52158f63bb78edd13):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 27 32 31 29 31 17 57]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 28 40 30 34 34 33 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 18 25 37 36 31 34 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 18 25 40 37 32 34 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 33 49 26 28 33 34 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 30 20 39 41 21 27 41]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 30 15 42 42 21 24 45]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 34 43 15 40 24 31 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 36 45 15 35 24 34 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 34 27 45 33 31 33 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 27 15 66 54 10 17 38]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 26 32 43 37 34 25 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 32 35 29 13 60 35 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 34 20 41 18 56 42 16]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 30 20 40 72 0 3 60]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 26 34 31 41 33 31 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 25 40 23 20 54 48 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 25 23 29 33 39 42 41]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 30 12 36 40 32 30 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 30 38 23 31 32 32 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 28 61 14 23 32 51 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [18 32 37 38 38 31 38 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [43 29 16 46 32 28 18 41]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [45 31 45 27 20 33 7 50]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [46 26 50 16 16 37 42 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [46 30 22 31 33 27 39 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [46 30 11 46 48 12 18 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [46 31 47 10 38 28 12 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 33 64 0 6 58 39 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 34 56 16 19 44 42 18]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 26 46 32 33 34 34 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 39 26 43 30 32 19 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 40 16 39 26 30 50 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 29 31 36 26 29 51 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 24 39 30 29 31 41 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 47 18 35 33 33 38 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 47 18 37 33 33 42 14]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [40 30 38 20 31 30 30 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [42 22 46 14 26 30 23 53]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 30 38 30 32 30 29 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [15 35 22 52 33 29 60 8]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 25 22 50 30 34 54 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 27 28 30 28 32 25 55]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 30 42 22 28 27 35 48]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 37 5 42 38 38 57 5]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 28 50 20 27 28 35 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 26 76 4 12 20 17 65]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 37 48 28 22 22 18 56]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [27 37 48 28 22 22 18 56]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.02s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot T-kv branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15600669?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15600669?buildTab=artifacts#/) on master @ [2ce93809d799bacc078f8a1d8e24a710dbff4d66](https://github.com/cockroachdb/cockroach/commits/2ce93809d799bacc078f8a1d8e24a710dbff4d66):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 28 32 27 27 43 24 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 28 25 35 50 25 27 42]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 29 36 26 41 38 30 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 32 41 29 25 48 33 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 30 32 43 29 38 31 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 23 23 49 41 17 30 48]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 25 37 32 23 40 26 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 33 50 17 16 56 25 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 31 34 39 34 37 26 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 27 16 55 57 15 27 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 28 31 35 36 33 29 29]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [8 7 12 7 23 5 9 13]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 31 41 29 20 52 30 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 33 19 44 58 17 30 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 27 14 47 68 7 31 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 20 29 38 44 26 31 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 42 39 29 25 41 28 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [18 51 19 33 44 25 33 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 28 27 32 46 23 34 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [57 7 50 23 31 31 30 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [51 26 52 23 13 36 30 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 50 37 29 34 25 27 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 36 28 37 51 22 28 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 19 32 28 33 43 35 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 39 30 27 26 47 36 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 39 30 29 42 29 34 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 30 29 35 48 17 26 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [42 18 30 34 24 55 20 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 41 31 31 11 60 36 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [15 50 32 30 34 36 44 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 38 31 28 46 20 31 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [54 14 30 26 29 41 16 47]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 32 30 26 24 43 31 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [15 57 30 27 35 33 34 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [47 34 30 29 40 21 28 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [57 24 28 29 23 41 30 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 45 28 28 12 52 38 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [10 50 30 28 55 14 39 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [48 21 29 27 65 8 29 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [57 10 28 27 40 40 26 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [49 9 28 31 33 49 29 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 28 29 33 30 43 36 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 36 35 33 31 33 28 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 38 27 27 30 39 39 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 38 26 27 27 41 40 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 31 32 29 40 19 30 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 20 43 39 51 10 16 52]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [16 39 33 37 24 50 18 40]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [16 39 33 37 24 50 18 40]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (257.51s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot T-kv branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15608844?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15608844?buildTab=artifacts#/) on master @ [ed40d3c996fae6efd323d94e8982d95732dc6e4e](https://github.com/cockroachdb/cockroach/commits/ed40d3c996fae6efd323d94e8982d95732dc6e4e):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 31 37 48 14 33 41 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 22 62 14 14 42 26 47]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 32 52 23 14 42 23 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 38 27 55 14 30 37 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 28 38 43 14 32 37 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 19 52 18 14 42 26 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 40 32 45 14 38 24 38]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [18 51 13 59 14 36 41 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 45 24 52 14 31 43 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 37 44 33 14 29 39 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 31 41 38 14 31 33 41]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 29 24 47 14 35 42 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 38 33 41 14 30 42 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 37 50 28 14 30 38 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 33 39 43 14 32 34 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 30 26 48 14 40 32 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 40 33 44 14 38 29 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 45 55 22 14 27 28 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 33 45 33 14 25 29 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [42 26 20 52 14 41 33 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 43 35 40 14 34 34 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [18 54 66 18 14 20 22 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 50 53 23 14 25 21 46]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [47 22 22 46 14 43 32 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 49 54 23 14 32 28 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [7 70 68 11 14 7 34 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 41 42 19 14 36 32 42]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [44 22 25 35 14 53 31 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [19 43 32 34 14 54 33 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [15 57 46 29 14 24 37 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [48 25 37 27 14 39 35 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [50 23 34 28 14 43 34 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 55 32 33 14 39 30 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 52 48 27 14 22 39 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [64 13 25 37 14 48 29 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [74 4 9 44 14 52 29 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 46 37 27 14 41 30 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 60 50 18 14 35 30 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 51 32 38 14 34 30 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 26 14 62 14 27 33 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [45 25 29 53 14 31 26 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 36 61 23 14 33 24 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 35 47 37 14 35 36 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 33 27 52 14 31 40 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 33 37 42 14 32 37 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 36 56 11 14 42 27 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 34 38 42 14 31 30 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 33 15 67 14 27 39 29]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [34 33 15 67 14 27 39 29]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (246.12s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15609334?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15609334?buildTab=artifacts#/) on master @ [ac32f09eca802de6883c52308c216edc2c4cf1d5](https://github.com/cockroachdb/cockroach/commits/ac32f09eca802de6883c52308c216edc2c4cf1d5):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 43 27 28 60 21 24 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 21 27 38 15 59 39 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 34 30 28 16 57 39 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [19 54 28 26 55 19 24 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [20 55 28 29 72 3 19 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 34 26 29 29 46 38 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 26 29 29 15 60 44 17]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [10 36 28 27 23 60 20 50]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [12 42 30 27 55 20 10 61]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 34 33 30 25 16 58 17]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 19 27 33 9 45 70 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [40 22 32 33 34 40 30 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 22 31 33 35 37 27 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 34 35 33 27 21 47 23]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 41 32 35 30 25 49 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 23 27 28 31 38 32 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [48 3 27 27 34 42 24 52]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 52 34 33 30 29 30 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [4 80 41 38 35 21 34 6]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 44 33 29 31 29 29 29]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [14 21 12 4 4 10 12 7]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 40 32 23 30 32 21 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 19 25 48 33 30 56 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 30 27 36 27 24 67 8]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [43 36 29 16 24 32 9 65]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [43 36 31 2 29 39 0 72]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [45 35 33 30 36 37 0 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [45 36 29 43 39 37 0 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 35 50 24 35 35 0 44]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 38 52 1 25 32 53 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 34 34 1 29 34 65 23]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 33 18 27 33 34 50 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 33 15 41 34 32 35 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 28 32 36 26 26 32 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [39 27 24 38 33 31 32 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 40 20 45 38 35 29 15]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [17 47 49 29 23 25 27 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 36 66 13 16 24 28 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [53 23 14 46 35 24 27 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 30 9 62 43 26 32 13]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [17 59 13 23 36 38 24 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 43 53 12 14 34 22 52]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [39 28 49 31 39 23 28 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 29 31 43 46 31 42 3]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 31 30 36 34 32 33 32]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [1 4 24 12 8 6 22 7]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 31 31 35 34 33 33 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 31 33 27 30 33 29 36]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [33 31 33 27 30 33 29 36]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.70s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15610827?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15610827?buildTab=artifacts#/) on master @ [e54d4a2655ea7b23ceb8dda114b0c01d8c52cafa](https://github.com/cockroachdb/cockroach/commits/e54d4a2655ea7b23ceb8dda114b0c01d8c52cafa):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 26 68 9 15 58 27 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 27 34 46 23 36 30 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 34 11 60 32 20 36 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 27 44 19 25 41 37 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 33 39 33 27 37 31 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 34 20 56 35 18 29 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 29 32 51 39 16 29 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 26 62 14 23 49 29 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 33 57 28 16 54 24 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 32 22 58 38 21 24 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 31 6 66 49 9 30 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 30 47 30 23 30 33 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 30 60 18 10 52 34 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 27 31 40 38 37 27 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 27 27 46 52 26 22 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 27 39 26 24 52 40 20]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 29 58 7 2 63 48 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 32 36 27 30 35 42 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [24 31 26 42 58 17 31 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 27 38 19 60 27 25 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 31 39 17 23 61 26 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 42 28 49 18 17 31 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [24 35 20 52 53 5 31 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 28 46 19 38 44 26 20]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 35 36 31 35 41 26 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 39 6 61 28 23 33 43]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 40 21 53 26 18 32 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 26 63 19 26 35 28 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 33 54 24 31 38 28 13]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 39 15 45 32 31 23 49]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 32 18 43 32 30 24 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 25 60 16 30 29 26 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [43 20 63 14 29 29 40 21]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 29 31 50 30 29 34 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 31 30 47 27 29 29 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 25 43 25 31 29 49 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 26 38 27 32 31 53 14]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 35 20 42 28 31 27 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 27 24 46 25 32 8 65]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 21 51 23 25 30 45 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 28 53 23 24 28 61 13]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [21 34 23 56 24 26 29 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 34 18 60 23 25 12 55]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 31 47 30 23 25 37 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 27 39 35 27 27 46 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 32 16 42 28 33 14 61]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 31 36 26 33 34 1 65]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 27 53 16 35 31 20 38]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [39 27 53 16 35 31 20 38]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (245.31s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15616551?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15616551?buildTab=artifacts#/) on master @ [6ce16f4a546444f8428a2412c4c64e177653d8b7](https://github.com/cockroachdb/cockroach/commits/6ce16f4a546444f8428a2412c4c64e177653d8b7):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [14 63 14 24 29 49 28 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [42 31 38 23 40 25 25 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [58 15 45 28 43 5 34 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 34 37 31 4 64 34 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 36 25 29 15 64 27 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [50 22 40 21 60 17 22 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 35 48 14 69 5 24 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [13 52 32 34 39 31 27 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 40 31 36 27 39 23 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [62 7 24 34 34 26 41 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 17 28 31 51 18 42 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [20 34 28 33 47 21 40 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [44 37 30 37 16 32 28 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [53 17 30 30 43 26 42 14]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 26 26 28 45 28 31 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [11 41 41 35 26 37 43 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [11 44 40 39 24 40 47 14]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [51 24 12 33 36 30 14 55]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [53 25 17 32 35 29 8 58]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 27 41 28 28 26 35 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 32 44 26 22 28 55 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 31 29 32 35 28 30 39]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 28 24 31 43 34 15 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 32 33 32 19 47 45 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 26 29 36 20 42 57 13]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 24 9 39 69 5 33 41]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 27 13 39 63 6 30 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 33 23 28 39 32 32 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 36 38 28 29 41 33 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [47 22 23 40 31 29 31 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [48 23 23 38 36 25 34 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [14 49 38 23 33 32 31 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [5 49 48 25 29 39 29 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [66 13 13 47 35 30 33 21]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [54 18 26 38 38 26 37 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 38 54 13 30 31 29 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 42 45 24 30 34 26 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [48 12 9 68 32 30 36 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [47 14 13 64 32 29 37 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 44 45 29 31 30 32 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [17 44 49 26 31 33 32 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 25 35 40 30 30 35 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 24 35 39 31 30 35 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 38 33 22 33 32 29 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 37 29 39 31 31 32 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 34 24 48 28 27 44 20]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 31 35 39 27 31 42 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 32 48 10 33 35 6 54]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [35 32 48 10 33 35 6 54]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (245.06s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15617253?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15617253?buildTab=artifacts#/) on master @ [f562d85d1533a4e103eb0199eda096d8c203b78b](https://github.com/cockroachdb/cockroach/commits/f562d85d1533a4e103eb0199eda096d8c203b78b):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 37 26 37 29 29 24 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 34 35 28 30 27 30 43]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 28 43 23 32 27 51 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 29 31 40 31 29 47 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 29 25 42 31 31 27 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 29 36 22 31 31 41 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 30 44 15 31 31 48 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 31 31 38 31 30 46 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 31 23 45 31 30 36 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 30 33 50 26 27 40 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 31 58 10 31 33 45 16]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 37 33 1 36 38 35 41]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 36 16 34 31 30 22 53]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 35 17 47 30 24 23 45]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 28 40 43 27 35 41 13]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 31 33 30 31 36 27 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 36 22 22 41 29 12 68]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 30 27 25 31 30 45 39]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 28 31 31 25 29 56 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 34 27 32 43 24 32 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 38 27 30 45 25 17 50]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 36 34 26 34 28 41 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [43 21 39 29 24 30 53 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 23 32 33 28 37 44 39]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 29 28 31 33 36 31 46]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [48 25 32 31 28 26 43 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [49 26 32 31 28 23 50 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 33 29 28 23 33 29 44]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [10 38 27 31 29 44 10 67]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 19 27 35 33 38 34 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [64 5 34 29 31 18 61 13]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 37 33 25 26 26 42 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [15 61 21 26 23 42 23 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 23 20 33 28 38 32 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [60 21 43 33 30 15 50 8]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 41 33 24 28 24 30 47]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 51 14 20 24 41 23 58]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 41 27 18 25 38 26 51]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 30 41 26 35 30 39 23]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 31 23 34 26 40 21 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 34 19 34 16 47 16 60]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 30 40 24 35 30 34 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 26 46 24 56 6 55 13]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 26 30 28 44 30 47 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 27 25 29 25 53 14 55]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 29 28 29 28 45 30 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 29 33 29 33 29 49 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 28 24 33 28 35 25 46]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [38 28 24 33 28 35 25 46]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15621098?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15621098?buildTab=artifacts#/) on master @ [b7d862cec226c54417cd2caab0aadf180ea96364](https://github.com/cockroachdb/cockroach/commits/b7d862cec226c54417cd2caab0aadf180ea96364):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 28 31 37 48 29 26 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 27 28 47 37 29 46 18]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 33 41 30 30 33 50 18]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 34 45 13 34 34 17 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 40 20 33 38 33 6 61]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 25 42 23 30 35 59 16]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 24 44 24 26 35 68 10]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 29 27 34 29 29 29 54]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 29 27 34 30 28 29 55]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 33 34 26 27 33 46 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 38 40 21 25 39 55 10]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 18 37 33 30 40 42 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 17 35 36 30 38 39 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 51 32 28 31 22 41 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 45 37 25 36 22 60 5]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 17 31 34 28 40 27 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 14 30 37 25 43 18 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 43 28 32 35 31 44 19]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 45 30 30 37 27 46 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 39 28 32 30 35 37 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 32 30 35 26 38 29 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 26 37 36 25 37 31 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 39 60 16 46 15 44 6]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 45 45 22 45 25 21 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 37 24 31 14 52 0 69]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 31 22 30 20 45 32 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 25 25 29 25 43 37 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 31 34 30 45 32 31 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 32 35 30 46 28 31 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 39 28 33 14 48 26 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 38 29 33 13 48 26 45]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 33 37 33 52 26 35 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 33 37 33 52 26 34 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 33 27 32 28 43 21 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 33 28 32 28 43 21 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 26 34 29 44 28 49 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 37 36 32 50 12 54 9]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 37 20 23 18 55 23 56]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 35 20 23 14 59 23 59]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 25 27 25 48 32 27 49]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 25 30 27 50 28 30 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 44 27 33 5 52 32 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 43 26 34 5 52 32 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 34 31 30 33 38 26 41]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 29 31 31 62 13 42 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 38 28 27 29 44 21 44]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 38 28 27 29 44 19 46]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 35 35 35 40 29 30 29]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [25 35 35 35 40 29 30 29]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.78s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15621714?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15621714?buildTab=artifacts#/) on master @ [d102f9db95fb91646a980334207894831c079a79](https://github.com/cockroachdb/cockroach/commits/d102f9db95fb91646a980334207894831c079a79):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 39 41 21 31 30 37 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 31 40 15 26 36 28 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [64 3 5 60 23 27 12 60]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 0 1 75 33 22 38 38]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 45 6 32 35 30 59 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 42 26 26 34 30 48 21]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 34 40 19 30 34 32 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 19 11 53 35 31 51 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 18 19 50 37 28 54 18]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [21 42 51 23 19 25 31 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [21 43 52 23 18 26 27 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 26 6 46 52 23 40 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 22 5 51 54 22 43 23]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 35 31 29 46 17 36 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 32 44 26 27 31 28 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 25 14 40 47 27 38 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 46 9 41 49 26 39 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [21 44 41 25 39 27 22 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [40 26 51 21 20 36 20 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [58 3 31 37 37 19 46 22]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [68 2 25 37 43 13 57 13]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [7 68 31 30 22 47 21 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [0 74 34 24 13 58 3 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [15 53 33 29 31 24 27 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 43 30 30 37 20 41 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 41 33 25 23 42 29 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 39 33 25 23 43 27 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 26 26 42 46 9 38 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 26 26 42 46 8 39 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 41 39 18 5 70 21 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 41 39 18 5 70 20 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 41 19 38 18 53 24 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 40 19 39 19 53 25 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [36 29 51 17 42 21 34 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 28 45 30 29 23 37 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 36 1 60 31 47 24 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 20 0 56 34 47 21 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [56 22 45 23 26 27 30 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [52 27 46 23 26 26 31 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [5 52 14 50 34 37 31 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [5 54 15 45 34 37 32 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [68 12 49 11 20 27 34 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [70 11 47 17 20 26 34 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 11 31 54 35 30 31 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 26 32 51 38 29 22 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 41 43 27 26 30 33 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 33 40 36 23 32 38 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [48 13 13 52 42 29 23 35]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [48 13 13 52 42 29 23 35]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.71s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15622451?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15622451?buildTab=artifacts#/) on master @ [d102f9db95fb91646a980334207894831c079a79](https://github.com/cockroachdb/cockroach/commits/d102f9db95fb91646a980334207894831c079a79):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 24 5 69 28 27 17 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 24 5 69 28 27 17 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 32 49 21 30 28 44 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 29 50 20 31 29 45 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 30 11 49 32 31 12 61]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 32 29 28 32 29 40 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 32 48 9 29 28 72 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 37 43 8 29 30 42 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 25 31 30 28 33 32 47]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 23 25 47 27 28 41 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 35 31 39 25 24 44 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 45 40 12 30 35 32 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 32 20 37 30 35 20 57]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 14 22 52 29 26 19 62]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [17 43 48 31 27 18 38 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [42 30 29 32 25 34 34 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [44 30 26 35 25 35 29 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [12 40 51 21 37 35 26 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [7 39 67 13 50 9 36 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 27 59 17 55 2 40 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 27 47 30 47 3 40 38]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 32 35 36 15 39 35 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [13 42 34 31 24 46 33 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 14 37 33 46 14 35 41]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [53 12 39 37 22 19 37 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 31 35 35 24 25 35 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 31 35 35 24 25 35 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 33 30 31 33 36 34 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [24 33 29 33 34 36 33 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [47 27 36 30 30 35 32 19]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [49 25 38 30 25 33 24 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 31 35 30 44 11 30 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [2 27 18 41 46 38 62 20]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 22 28 27 39 43 41 17]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [62 20 39 24 19 46 24 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [17 44 30 33 37 33 34 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [2 48 18 36 57 24 47 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [2 31 49 32 35 38 47 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [58 21 51 13 1 54 4 52]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [57 23 31 21 36 36 2 51]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [43 32 31 29 38 36 2 48]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 37 27 35 39 27 39 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 28 37 31 29 28 44 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [42 27 37 27 30 30 26 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 41 30 37 34 31 22 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [46 24 31 37 34 31 27 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [59 6 35 21 32 34 37 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [45 35 32 39 31 22 20 32]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [45 35 32 39 31 22 20 32]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.26s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15622648?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15622648?buildTab=artifacts#/) on master @ [d102f9db95fb91646a980334207894831c079a79](https://github.com/cockroachdb/cockroach/commits/d102f9db95fb91646a980334207894831c079a79):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 45 45 18 12 43 20 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 41 35 34 11 34 35 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 36 33 41 11 29 41 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 24 49 31 40 23 32 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [21 39 66 13 50 30 5 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 22 44 26 40 29 36 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [43 18 23 51 18 27 64 12]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 35 22 43 31 28 56 18]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [10 53 30 13 39 29 17 63]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 41 31 24 21 35 17 60]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [61 8 20 53 17 28 43 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [54 8 33 37 37 26 38 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [13 64 62 10 41 32 6 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [13 64 56 10 41 32 13 29]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [46 13 4 59 29 31 54 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [39 22 25 41 24 28 54 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 36 63 11 29 27 26 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 33 47 27 30 28 31 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 28 33 43 29 29 39 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 26 32 35 31 32 30 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [43 32 31 22 31 32 23 42]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 38 35 23 29 40 21 44]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [16 18 29 53 25 36 44 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 15 29 47 27 34 42 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 35 31 25 37 31 32 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 25 28 39 22 37 41 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 25 28 39 22 37 44 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 36 30 25 48 24 35 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 36 32 19 50 25 32 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 22 13 58 34 30 34 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 19 11 66 26 35 54 13]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 38 19 45 30 29 46 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 39 25 39 31 26 42 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [42 21 50 26 26 27 31 35]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [50 20 40 25 25 35 38 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 49 29 32 34 32 25 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [16 61 22 36 36 28 18 40]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 31 27 32 36 30 32 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [61 6 49 16 30 30 56 7]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [43 6 42 31 34 32 57 14]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [3 54 24 46 32 31 4 63]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [3 57 22 39 34 32 3 67]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 35 29 25 27 31 49 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [40 34 33 20 27 31 52 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 25 20 48 31 27 21 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 37 57 8 30 29 50 15]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 40 64 0 32 31 54 2]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 24 26 33 32 38 43 25]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [36 24 26 33 32 38 43 25]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.08s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15626029?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15626029?buildTab=artifacts#/) on master @ [a983d2291a0d17e9993104aaa6b07ec0538230ab](https://github.com/cockroachdb/cockroach/commits/a983d2291a0d17e9993104aaa6b07ec0538230ab):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [44 26 46 25 41 23 23 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 32 26 35 35 28 54 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [6 55 18 45 14 36 73 11]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 31 17 40 24 30 57 23]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 22 28 28 41 24 29 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 42 33 28 21 45 41 23]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 37 37 30 11 52 46 11]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [61 10 29 33 75 6 21 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [52 4 15 30 79 1 3 69]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [40 21 23 20 41 36 17 58]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 44 26 23 11 59 54 14]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [20 49 23 27 46 23 62 8]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [28 29 23 30 52 18 31 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 22 25 30 41 30 29 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 37 30 30 25 38 29 33]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 39 32 33 34 26 33 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [46 26 33 35 37 19 34 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [56 14 28 23 14 45 31 41]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 29 28 24 17 46 34 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [3 55 35 25 40 32 36 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [3 52 36 25 45 29 38 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 13 38 45 14 46 33 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [59 10 33 44 4 40 28 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [45 32 19 18 65 9 30 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 45 37 17 75 1 29 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 28 18 46 36 38 25 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [42 28 9 62 17 45 22 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 39 53 25 34 29 32 23]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 32 67 15 43 27 38 15]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [43 16 33 42 36 32 24 33]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [38 16 23 49 29 38 18 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 45 28 38 25 31 19 49]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [21 50 32 27 25 29 44 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [48 19 34 44 36 42 20 13]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 34 28 44 23 39 4 58]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [14 57 28 23 29 29 74 5]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [8 70 30 14 47 10 79 0]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [40 39 20 48 32 30 49 0]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 36 22 48 32 30 49 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 39 56 23 37 40 18 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [39 39 58 25 40 43 12 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 35 15 62 21 17 68 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 35 14 62 21 17 69 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 36 47 19 32 41 34 6]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 29 57 5 35 43 4 62]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 28 28 42 34 36 18 42]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 34 5 80 35 20 38 9]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 35 33 39 24 34 30 40]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [22 35 33 39 24 34 30 40]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.93s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15628777?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15628777?buildTab=artifacts#/) on master @ [f5e65c553608fe6e1c0db6b2a8b279b33790ce2e](https://github.com/cockroachdb/cockroach/commits/f5e65c553608fe6e1c0db6b2a8b279b33790ce2e):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [54 27 38 35 25 24 43 10]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [12 43 37 41 38 25 55 4]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [8 29 37 40 40 28 24 53]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [40 20 32 20 25 33 10 76]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [12 62 21 33 26 23 58 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [5 66 18 38 31 28 66 2]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [40 32 25 30 28 28 27 47]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [44 31 29 24 25 29 20 56]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [54 21 30 21 22 24 27 61]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [50 24 28 23 24 21 41 48]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 35 33 23 29 31 39 37]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [18 32 49 19 31 32 9 63]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 25 37 26 26 29 67 15]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 32 13 46 23 29 79 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [18 33 50 20 27 27 31 50]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [10 34 54 6 39 27 23 62]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [40 21 40 20 32 28 31 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [48 19 28 36 26 27 35 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 40 35 44 32 27 27 32]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [20 43 42 35 34 28 22 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 41 40 21 33 30 35 30]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 36 35 29 29 29 39 29]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [5 10 16 5 12 5 10 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [19 36 22 62 31 27 11 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [23 38 41 43 31 26 5 51]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [45 20 27 29 31 30 56 20]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [47 28 17 25 25 37 78 1]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 51 18 24 31 31 73 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 56 45 23 43 33 30 0]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [42 25 29 39 47 33 42 0]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [46 25 28 39 35 33 43 9]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [15 41 38 25 23 33 33 51]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [12 51 44 20 28 29 31 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [50 28 25 45 42 22 32 14]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [50 23 19 49 36 26 35 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 38 57 11 19 41 18 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 39 61 0 16 44 16 52]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [54 16 30 0 50 26 50 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [59 19 15 0 58 22 53 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [14 45 50 0 33 37 35 44]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [15 54 60 0 12 47 15 55]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 32 32 40 27 35 23 43]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 26 25 47 34 29 30 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [24 30 50 24 27 32 25 46]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 31 56 16 22 34 22 50]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 26 15 48 41 18 50 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 26 15 48 41 18 50 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 28 36 32 22 47 40 26]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 29 replicas on all stores): [28 28 36 32 22 47 40 26]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.40s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15630408?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15630408?buildTab=artifacts#/) on master @ [daf384afdcd7b6968414e3d4826503a295bc0590](https://github.com/cockroachdb/cockroach/commits/daf384afdcd7b6968414e3d4826503a295bc0590):


```
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 27 24 36 26 37 67 14]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 25 34 23 31 35 32 49]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 38 44 20 44 12 11 58]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [32 43 22 37 19 29 41 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 28 22 36 8 56 55 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 33 52 18 62 12 23 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 32 62 13 72 4 15 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 31 40 34 46 27 20 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 28 30 39 38 39 31 20]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 34 36 17 25 37 36 45]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 35 49 16 34 8 13 62]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [37 39 17 62 38 4 38 22]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 21 4 70 7 74 53 3]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [16 14 21 57 7 77 52 14]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [15 14 35 42 21 55 38 38]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [25 31 35 34 34 37 34 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 34 36 15 58 16 35 24]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 26 18 56 26 40 25 36]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 29 11 66 10 55 22 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [23 30 37 36 42 35 23 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 25 62 13 58 20 37 8]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 18 31 39 34 26 22 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 16 26 47 28 37 19 52]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [13 50 35 37 33 35 21 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 35 38 28 35 29 41 15]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [42 34 30 30 30 30 23 37]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 30 31 33 28 33 7 65]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [17 32 37 27 38 27 41 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 30 34 24 38 27 63 11]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [41 29 26 29 28 30 42 34]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 31 27 31 27 32 21 58]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [17 32 34 30 29 30 49 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [24 31 35 30 30 29 54 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [63 14 20 40 30 35 4 48]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [50 33 23 40 30 34 1 47]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [19 39 41 26 28 27 60 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [37 22 42 30 26 24 64 14]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [52 18 27 40 28 34 9 50]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 36 14 47 27 43 1 56]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [14 60 34 37 30 42 1 40]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [15 62 37 36 36 39 1 32]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 55 37 32 36 37 1 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 27 25 49 36 35 1 48]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 41 56 24 37 38 1 28]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 41 46 40 37 39 1 20]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 30 12 49 30 23 59 21]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 35 21 44 24 17 63 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 32 43 25 26 31 42 28]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [29 32 43 25 26 31 42 28]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.86s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

kv/kvserver.TestReplicateQueueRebalanceMultiStore [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15632739?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/15632739?buildTab=artifacts#/) on master @ [718b6cad0408b5d785449f2e2a189f1a8e6ff1ae](https://github.com/cockroachdb/cockroach/commits/718b6cad0408b5d785449f2e2a189f1a8e6ff1ae):


```
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 35 60 12 41 18 30 31]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [30 28 54 27 28 34 29 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 22 22 61 20 44 29 26]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [27 30 34 47 18 46 28 28]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [20 47 45 15 51 12 28 36]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [42 34 36 12 55 12 28 39]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [43 18 7 69 18 43 32 27]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [34 32 24 57 15 41 30 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [27 41 51 36 24 27 25 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 25 35 27 35 30 33 29]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [48 9 24 48 32 32 24 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [24 52 27 38 35 23 31 26]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [19 58 33 32 43 19 37 15]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 42 35 30 43 22 43 17]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [26 42 35 30 43 22 42 18]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [31 30 31 29 40 28 31 38]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 32 32 30 29 35 28 36]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [5 11 16 24 3 9 9 7]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 33 32 34 22 33 34 35]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [32 33 34 34 18 33 36 34]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [29 28 28 33 49 28 30 28]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [8 8 12 21 15 7 7 6]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 28 27 32 49 28 28 31]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [30 34 38 28 26 38 38 25]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 35 41 29 13 41 42 25]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [33 31 28 40 32 23 42 30]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [31 33 24 37 60 9 19 43]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [35 32 34 23 70 0 36 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [34 29 40 21 31 33 47 19]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [35 29 29 33 14 55 51 12]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [25 29 11 42 38 44 10 56]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [22 28 46 18 35 44 4 61]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 29 64 14 20 37 57 1]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [26 29 36 31 27 36 45 27]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [22 29 4 54 41 27 2 74]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 31 41 31 43 28 2 54]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [36 30 67 6 23 30 54 9]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [38 30 67 6 19 31 64 1]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 29 31 45 25 32 61 1]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [33 31 26 50 26 31 59 1]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 31 33 39 30 29 34 34]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [7 7 16 10 5 8 23 8]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [29 30 34 36 32 29 34 35]
    replicate_queue_test.go:344: current lease state (want at least 7 leases on all stores): [7 6 15 9 5 7 22 13]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [39 33 32 25 30 37 38 24]
    replicate_queue_test.go:333: current replica state (want at least 29 replicas on all stores): [28 33 33 37 33 33 42 21]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [17 31 32 39 27 32 30 46]
    replicate_queue_test.go:333: current replica state (want at least 28 replicas on all stores): [41 24 29 17 33 25 24 62]
    replicate_queue_test.go:330: condition failed to evaluate within 4m0s: from replicate_queue_test.go:336: not balanced (want at least 28 replicas on all stores): [41 24 29 17 33 25 24 62]
    --- FAIL: TestReplicateQueueRebalanceMultiStore/multi-store (244.34s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #125450 kv/kvserver: TestReplicateQueueRebalanceMultiStore failed [C-test-failure O-robot P-1 T-kv X-duplicate branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestReplicateQueueRebalanceMultiStore.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>

--- a/pkg/kv/kvserver/replicate_queue_test.go
+++ b/pkg/kv/kvserver/replicate_queue_test.go
@@ -249,6 +249,11 @@ func TestReplicateQueueRebalanceMultiStore(t *testing.T) {
 				st := server.ClusterSettings()
 				st.Manual.Store(true)
 				allocatorimpl.LeaseRebalanceThreshold.Override(ctx, &st.SV, leaseRebalanceThreshold)
+				// We speed up replicate queue processing (scan min/max idle) time,
+				// this causes actions to occur more frequently than in practice and
+				// ultimately this test will fail unless we correspondingly increase
+				// the max store gossip frequency.
+				kvserver.MaxStoreGossipFrequency.Override(ctx, &st.SV, 0)
 			}
 
 			// Add a few ranges per store.

